lib/mix/tasks/docs.ex

```diff
@@ -361,6 +361,27 @@ defmodule Mix.Tasks.Docs do
         "get-started" => "quickstart"
       }
 
+  ## Customizing Extras
+
+  - `:title` - The title of the extra page. If not provided, the title will be inferred from the filename.
+  - `:filename` - The name of the generated file. If not provided, the filename will be inferred from the source file.
+  - `:source` - The source file of the extra page. This is useful if you want to customize the filename or title but keep the source file unchanged.
```

Comment:
Let's make sure to keep these lines shorter, wrapping around 98 (Elixir's length)